src/EntityFramework6.Npgsql/NpgsqlMigrationSqlGenerator.cs

```diff
@@ -500,18 +500,50 @@ protected virtual void Convert(DropPrimaryKeyOperation dropPrimaryKeyOperation)
         #region Misc functions
 
         /// <summary>
+        /// Quotes an identifier for Postgres SQL and appends it to a string builder
```

Comment:
nitpick: "string builder" -> <see cref="StringBuilder" /> would be nice to get links.